common/src/main/resources/data/conditional/recipes/farm_and_charm/drying/dried_corn_from_ce.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "doapi:conditional",
+  "recipes": [
+    {
+      "conditions": [
+        {
+          "type": "doapi:not",
+          "value": {
+            "type": "forge:mod_loaded",
+            "modid": "brewery"
+          }
+        }
+      ],
+      "recipe": {
+        "type": "farm_and_charm:drying",
+        "ingredient": {
+          "item": "farm_and_charm:corn"
+        },
+        "recipe_type": "MEAT",
```

Comment:
Should the recipe_type be "MEAT" ?

Reply:
I copied this directly from Brewery and tweaked it. It's MEAT there also, not sure what the other values are. But hey, at least we know meat is valid in the silo. Jerky expansion next?

Reply:
Doesn't look like it matters, at least couldn't find a use in the code
Might be related to the recipe book, but if it is it's not following the vanilla format and it's not followed anyway